src/Microsoft.SymbolStore/SymbolStores/HttpSymbolStore.cs

```diff
@@ -16,12 +16,12 @@
 namespace Microsoft.SymbolStore.SymbolStores
 {
     /// <summary>
-    /// Basic http symbol store. The request can be authentication with a PAT for VSTS symbol stores.
+    /// Basic http symbol store. The request can be authentication with a PAT or bearer token for Azure symbol stores.
     /// </summary>
     public class HttpSymbolStore : SymbolStore
     {
         private readonly HttpClient _client;
-        private readonly HttpClient _authenticatedClient;
+        private readonly Func<CancellationToken, Task<AuthenticationHeaderValue>> _authenticationFunc;
```

Comment:
```suggestion
        private readonly Func<CancellationToken, ValueTask<AuthenticationHeaderValue>> _authenticationFunc;
```

Reply:
To not penalize PAT usage which is fully synchronous,

Reply:
I will have to add a reference to System.Threading.Tasks.Extensions.dll to use ValueTask since this assembly is netstandard2.0.  Is that ok?